changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 10.3.0 - xxxx-xx-xx =
+* Dev - Deprecates and replaces PRBs classes with ECE equivalents
```

Comment:
Nit: I think we should use longer names here rather than the acronyms.
```suggestion
* Dev - Deprecates and replaces Payment Request Button classes with Express Checkout equivalents
```

Reply:
Actually I think this change may need to be described as an `Update` due to the deprecation. This is something we should flag as much as possible. I think it should also get an entry in the "Compatibility Notes" section of the main readme.txt file.